scripts/make-env

```diff
@@ -31,10 +33,15 @@ randstr() {
 }
 
 b64encode() {
-  cat "$@" | base64 --wrap=0 2>/dev/null || cat "$@" | base64 --break=0
+    echo "$@" | base64 --wrap=0 2>/dev/null || echo "$@" | base64 --break=0 2>/dev/null
 }
 
-VPN_CONFIG=$(b64encode <<STR
+b64file() {
+  local FILE=$(awk 'FNR==1{print ""}1' $*)
+  b64encode "$FILE"
```

Comment:
Why use `awk` here?

```sh
b64file() {
  b64encode "$(cat "$@")"
}
```

Reply:
I was having issues with `cat` not putting a line ending after each file. I will try and retest it

Reply:
hmm, yeah cat will only print the files as-in (no extra endings) but everything _should_ be ended properly :S